.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
About squashing : the goal is to have a clean git log in the end (avoiding unnecessary commits).
In our case, if you commit the removal of the line in .gitignore, you will have 2 commits in the log, that would end up in the upstream log if the PR is merged.
Squashing means merging these 2 commits, so that to hide the details of this line added and removed.
You can easily do that in your local repo with command `git rebase -i master`. It will open a text editor with 2 lines (one for each commit). For each commit, you'll be able to say what you want to do with it (by replacing the command at the beginning of the line). In our case, you'll want to put "f" (for "fixup") at the beginning of your second commit. It should result in a single commit in your log, merging the 2. You can check with `git log`
After that, you will have to push to github again with a `--force` option. It will update the PR with this single commit.